rcp-stripe-elements.php

```diff
@@ -45,3 +45,56 @@ function rcp_elements_register_stripe_elements_gateway( $gateways ) {
 	return $gateways;
 }
 add_filter( 'rcp_payment_gateways', 'rcp_elements_register_stripe_elements_gateway' );
+
+/**
+ * Custom path for RCP template overrides
+ *
+ * @param array  $template_stack
+ * @param string $template_names
+ *
+ * @return array $template_stack
+ */
+function rcp_elements_custom_template_path( $template_stack, $template_names ) {
+	$template_stack[] = trailingslashit( plugin_dir_path( __FILE__ ) ) . 'templates/';
+
+	return $template_stack;
+}
+add_filter( 'rcp_template_stack', 'rcp_elements_custom_template_path', 10, 2 );
+
+/**
+ * Conditionally loads Stripe Elements JS for the `update card` page
+ *
+ * @return void
+ */
+function rcp_elements_load_scripts() {
+
+	// Bail early if scripts shouldn't be loaded
+	if ( rcp_elements_is_update_card_page() === false ) {
+		return;
+	}
+
+	$gateway = new RCP_Payment_Gateway_Stripe_Elements();
+	$gateway->scripts();
+}
+add_action( 'wp_enqueue_scripts', 'rcp_elements_load_scripts', 10, 0 );
+
+/**
+ * Checks if the current page is the RCP update card page
+ *
+ * @return bool $ret
+ */
+function rcp_elements_is_update_card_page() {
```

Comment:
Only suggestion would be to update this to just return in each conditional and then return again at the end, instead of doing the variable assigning.

Reply:
Thanks updated